src/plugins/workspace/public/components/workspace_form/workspace_detail_form.tsx

```diff
@@ -66,79 +99,172 @@ export const WorkspaceDetailForm = (props: WorkspaceFormProps) => {
     handleDescriptionChange,
   } = useWorkspaceForm(props);
 
-  const isDashboardAdmin = application?.capabilities?.dashboards?.isDashboardAdmin ?? false;
+  useEffect(() => {
+    getNumberOfChanges(numberOfChanges);
```

Comment:
getNumberOfChanges seems more like an util function, we do not need to pass it through props.